src/keychain/index.js

```diff
@@ -174,7 +176,7 @@ class Keychain {
    * @param {int} size - The key size in bits.
```

Comment:
default value and optional?

Reply:
I updated this to be optional and updated the logic to remove the default. 0 doesn't really make sense, so now this just checks the size when it matters (when type is rsa). Also updated the api docs to reflect this.

Reply:
That makes sense, cool!